src/Serialization/ToricGeometry.jl

```diff
@@ -1,13 +1,30 @@
 ################################################################################
 # Toric varieties
 @register_serialization_type AffineNormalToricVariety uses_id
-@register_serialization_type NormalToricVariety uses_id
 
-function save_object(s::SerializerState, ntv::NormalToricVarietyType)
-  save_object(s, ntv.polymakeNTV)
+@register_serialization_type NormalToricVariety uses_id [:cox_ring]
+
+
+function save_object(s::SerializerState, ntv::T) where T <: NormalToricVarietyType
+  attrs = attrs_list(s, T)
+
+  if !isnothing(attrs) && any([has_attribute(ntv, attr) for attr in attrs])
```

Comment:
`attrs` will never be `nothing`, I think you need `isempty` for what you had here earlier.

you could, in theory, just save NTV from now on always as a dicts with `pm_data` and attrs, and only handle the old format in loading. this would make the saving already simpler, and it would not double the check for the presence of any attribute with https://github.com/oscar-system/Oscar.jl/pull/4020/files#r1723508786

Reply:
yes, I think the check here instead makes sense, this would then altert he toric variety files and this will introduce a breaking change, i'll need to write an upgrade script + probably upgrade martin's artifacts for f theory.

Reply:
cf https://github.com/oscar-system/Oscar.jl/pull/4020#discussion_r1721493505
tldr: I am not involved in NTV, so feel free to ignore my comments about that and instead talk to the people having a say there :)